src/client-config.prod.js

```diff
@@ -15,10 +15,12 @@ window.config = {
   AUTH_URL: 'https://gateway.{{hostname}}/auth/',
   MINIO_BUCKET: 'ocrvs',
   COUNTRY_CONFIG_URL: 'https://countryconfig.{{hostname}}',
-  // Country code in uppercase ALPHA-3 format
-  COUNTRY: 'FAR',
-  LANGUAGES: 'en,fr',
-  SENTRY: '{{sentry}}',
+  // Country code in uppercase ALPHA-3 format// Country code in uppercase ALPHA-3 format
+  COUNTRY: 'MDG', // MDG
+  AVAILABLE_LANGUAGES_SELECT: 'fr:Français,en:English,mg:Malagasy',
```

Comment:
`AVAILABLE_LANGUAGES_SELECT` was this deprecated on 1.4?

Reply:
Great catch! will remove it